modules/functions.test.js

```diff
@@ -0,0 +1,75 @@
+import { JSDOM } from 'jsdom';
+import { LocalStorage } from 'node-localstorage';
+import {
+  removeTask, getStorage, addList, tasks, taskEntry,
+} from './functions.js';
+
+const localStorage = new LocalStorage('./localstorage');
+
+describe('removeTask', () => {
+  let cleanup;
+
+  beforeEach(() => {
+    const dom = new JSDOM('<!DOCTYPE html><div id="container"></div>', {
+      url: 'http://localhost',
+    });
+    global.document = dom.window.document;
```

Comment:
- [ ]  Great job writing the test for add and remove functionality, :+1: :100: I think there are some improvements you need to make here before moving to the next milestone, :+1: 
- [ ] Currently, your tests are failing, it's giving an error in the console, I'll suggest you run the test one more time and try to fix the error following the suggestion from the console.
- Screenshot For reference:

![Screenshot from 2023-05-10 20-17-52](https://github.com/binodbhusal/ToDoList2/assets/107632780/4b222b1f-1747-409f-b78b-9ace12cc3d09)